cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -191,15 +191,36 @@ func checkForPoolExistence(cStorPool *apis.CStorPool, blockDeviceList []string)
 	return strings.Contains(stdoutStderr, string(PoolPrefix)+string(cStorPool.ObjectMeta.UID))
 }
 
+// checkIfPresent is to check if search string is present in array of string.
+func checkIfPresent(arrStr []string, searchStr string) bool {
```

Comment:
I think this is repetitive function we are already having this kind of func in `pkg/util/util.go`